telemetry/pings.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# Schema
+$schema: moz://mozilla.org/schemas/glean/pings/2-0-0
+
+page:
```

Comment:
Your use-case can be much simpler! No need for pings.yaml, just need metrics.yaml with _events_. Other properties can be attached to your events as you see fit.